### DISCUSSION TITLE
The discoverability of the project

### DISCUSSION THREAD

**Discussion by tissole** [aahnik/tgcf#203] [General]
I tried to search on Github using the following terms:
- Telegram forward
- Telegram forwarder
- Telegram clone
- Telegram copy

**Your script doesn't appear in the first 200 projects. Even after searching for most starred projects your program doesn't show in the 20 pages of search. It shows projects with no stars and even ones that are older than 4-5 years but not yours**.

Your project only shows when you search for "Telegram forwarding". So it's massive room for improvement here. Try adding the following tags:
-Telegram forward bot
-Telegram message forward
-Telegram message forwarder
-Telegram channel forwarder
-Telegram forwader bot
-Telegram forwarding bot
-Telegram copy bot
-Telegram channel copy
-Telegram duplicate chats
-Telegram channel duplicator
-Telegram duplicator bot
-Telegram auto-forward bot
-Telegram auto-forwarding bot
-Telegram auto-copy bot
-Telegram auto duplicator
-Telegram automating message bot
-Telegram sync bot
-Telegram channel sync
-Telegram chat sync
-Telegram backup bot
-Telegram channel backup

**Comment by aahnik** [comment]
Thanks a lot, @tissole, this is a very important issue.
